charts/ingress-nginx/templates/controller-ingressclass.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.controller.ingressClassResource.enabled -}}
```

Comment:
As IngressClass was introduced in kube 1.18, should we have a check for only adding this if the kube api 1.18 or above?
Something like `{{- if and (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) (.Values.controller.ingressClassResource.enabled) -}}`?

I see this is checked in other places:
https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/templates/_helpers.tpl#L106-L115
{{- if semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion -}}
Capabilities.KubeVersion.GitVersion

Reply:
Absolutely. I did not know this is possible via Helm.